### PR TITLE
Add support for Lonsonho X702A

### PR DESCRIPTION
**Pull request by dimich99**
Added support for [Lonsonho X702A](https://www.aliexpress.com/item/4001138076397.html) (2-gang with neutral).
Added LED backlight settings
Added power_on_behavior for each channel
image

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -1927,12 +1927,34 @@ const definitions: Definition[] = [
...
     {
         zigbeeModel: ['TS0002'],
         model: 'TS0002',
         vendor: 'TuYa',
         description: '2 gang switch',
-        whiteLabel: [{vendor: 'Zemismart', model: 'ZM-CSW002-D_switch'}, {vendor: 'Lonsonho', model: 'X702'},
+        whiteLabel: [
+            {vendor: 'Zemismart', model: 'ZM-CSW002-D_switch'},
+            {vendor: 'Lonsonho', model: 'X702'},
+            {vendor: 'Lonsonho', model: 'X702A'},
```
Comment (Koenkk) [inline src/devices/tuya.ts:1957]: Why was this added here? X702A = TS0002_switch_module_3, not TS0002, right?

Comment (dimich99) [reply src/devices/tuya.ts:1957]: Should I remove it from there and create separate zigbeeModel section or just remove?

Comment (Koenkk) [reply src/devices/tuya.ts:1957]: Just remove (because it is already under TS0002_switch_module_3)

**Comment by Koenkk** [comment]
Thanks!
